galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.selenium.manager/src/main/java/dev/galasa/selenium/ISeleniumManager.java

```diff
@@ -9,18 +9,21 @@
 
     /**
      * Allocate a new WebPage
+     * @throws SeleniumManagerException
      */
-    public IWebPage allocateWebPage();
+    public IWebPage allocateWebPage() throws SeleniumManagerException;
 
     /**
      * Allocate a new WebPage for a provided URL
+     * @throws SeleniumManagerException
      */
-    public IWebPage allocateWebPage(String url);
+    public IWebPage allocateWebPage(String url) throws SeleniumManagerException;
 
     /**
      * Allocate a new WebPage for a provided URL and Tag
+     * @throws SeleniumManagerException
      */
-    public IWebPage allocateWebPage(String url, String tag);
+    public IWebPage allocateWebPage(String url, String tag) throws SeleniumManagerException;
```

Comment:
Can you explain the purpose of a URL and a tag please?

Reply:
Removed tag as useless until SPI is implemented